xchange-huobi/src/main/java/org/knowm/xchange/huobi/HuobiUtils.java

```diff
@@ -59,7 +62,8 @@ public static void setHuobiAssetPairs(HuobiAssetPair[] huobiAssetPairs) {
   public static Currency translateHuobiCurrencyCode(String currencyIn) {
     Currency currencyOut = assetMap.get(currencyIn);
     if (currencyOut == null) {
-      throw new ExchangeException("Huobi does not support the currency code " + currencyIn);
+      logger.error("Huobi does not support the currency code " + currencyIn);
+      return null;
```

Comment:
and the returned null value will go to?

Reply:
I added this fix after Huobi introduced new currency that was not yet updated in Xchange project, leading to failure of Huobi exchange while loading, instead just logging the problem.

No use to prevent the whole loading process with exception when only 1 currency is problematic. 

I fixed setHuobiAssetPairs as well, so the null will be checked and ignored there.